exe/kubernetes-deploy

```diff
@@ -294,22 +289,33 @@ class KubernetesDeploy
   end
 
   class KubernetesResource
-    extend ActiveSupport::DescendantsTracker
```

Comment:
We only need to care about direct descendants of this one class, so I believe what's below is all we need of https://github.com/rails/rails/blob/master/activesupport/lib/active_support/descendants_tracker.rb. However, please scrutinize.